src/components/Header/Header.css

```diff
@@ -0,0 +1,51 @@
+@import "../../App.css";
+
+.header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
```

Comment:
You can add this as a utility in your app.css for reusabilty

Reply:
I'll check that out, thank you!